packages/web/src/components/search/DataSearch.js

```diff
@@ -191,9 +190,9 @@ class DataSearch extends Component {
 	// returns size and aggs property
 	getBasicQueryOptions = () => {
 		const { aggregationField, size } = this.props;
-		let queryOptions = { size };
+		const queryOptions = { size };
 		if (aggregationField) {
-			queryOptions = getCompositeAggsQuery({ ...queryOptions }, this.props, null, true);
+			queryOptions.aggs = getCompositeAggsQuery({}, this.props, null, true).aggs;
```

Comment:
@ShahAnuj2610 Why did you change this logic?
You're reading the function value without ensuring that the function returns an object which can break the app.

Reply:
https://github.com/appbaseio/reactivecore/blob/01b710f1862dbb8101919bee5c3d6f143d4c29fb/src/utils/helper.js#L450 => it returns an object